client/src/components/generic/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+
+type DropdownProps = React.SelectHTMLAttributes<HTMLSelectElement>
+
+const Dropdown = ({ ...props }: DropdownProps) => {
+  return (
+    <select
```

Comment:
is there some way to change the dropdown arrow? https://stackoverflow.com/a/42087251

Reply:
I added code to change the dropdown arrow but it made the text disappear. Any idea on what could be causing this?

Reply:
Maybe have a play around with inspect element to see if anything is covering it